buildIncremental.fsx

```diff
@@ -16,6 +16,16 @@ module IncrementalTests =
     | Linux
     | All
 
+    type Runtime =
+    | NetCore
+    | Net
+
+    let SkippedTest name runtime =
+        match (name, runtime) with
+            | (EndsWith "Sqlite.Tests.csproj", NetCore) -> false
```

Comment:
At the moment, packages with native dependencies are broken on .NET Core. Known issue with latest SDK.